src/main/java/org/bookstore/controller/BookController.java

```diff
@@ -15,35 +20,42 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Books management",description = "Endpoints for book management")
+@Validated
 @RequiredArgsConstructor
 @RestController
 @RequestMapping(value = "/books")
 public class BookController {
     private final BookService bookService;
 
+    @Operation(summary = "Getting all books")
     @GetMapping
-    List<BookDto> getAll() {
-        return bookService.findAll();
+    List<BookDto> getAll(Pageable pageable) {
```

Comment:
think of using `@PageableDefault` in case client did not specify any pagination parameters